shared/ui-components/src/locales/general.json

```diff
@@ -2,14 +2,12 @@
   "application": {
     "form": {
       "errors": {
-        "pleaseEnterZipCode": "Please enter your Zipcode",
         "selectOption": "Please select one of the options above.",
         "householdTooBig": "Your household size is too big.",
         "householdTooSmall": "Your household size is too small."
       },
-      "radio": {
-        "no": "No",
-        "yes": "Yes"
+      "general": {
+        "saveAndReturn": "Save & return to review"
```

Comment:
Maybe "Save & Return to Review"?

Reply:
Fine by me, but I'll let @slowbot weigh in